src/app/search/state/search.service.ts

```diff
@@ -644,9 +644,9 @@ export class SearchService {
    */
   saveCurrentTabAndClear(index: number) {
     if (index === SearchService.WORKFLOWS_TAB_INDEX) {
-      this.router.navigateByUrl('search?entryType=workflows&searchMode=Files');
+      this.router.navigateByUrl('search?entryType=workflows&searchMode=files');
```

Comment:
I looked and it's still not clear to me what searchMode actually does. It looks like it can be `files` or `description` (and I digress, but the type [here](https://github.com/dockstore/dockstore-ui2/blob/23d6e519f214dfaec4b9476a0ece63129a96a04a/src/app/shared/models/AdvancedSearchObject.ts#L22) should so indicate, although the type there wouldn't have fixed this anyway).

No action required on this comment.